src/components/molecules/ScheduleEvent/ScheduleEvent.tsx

```diff
@@ -84,12 +81,11 @@ export const ScheduleEvent: React.FC<ScheduleEventProps> = ({
           <div className="ScheduleEvent__title">{event.name}</div>
```

Comment:
@sunny-viktoryia this is the section I was referring to (where the prior reordering is now reverted). Part of the issue is that if events are very short, the title doesn't display, and is only visible from the event modal. Not a huge problem, but something that might be worth finding a fix for in the future